ci/blackbox/rdsbroker_test.go

```diff
@@ -387,3 +389,34 @@ func permissionsTest(databaseURI string) error {
 
 	return nil
 }
+
+func postgresExtensionsTest(databaseURI string) error {
+	db, err := openConnection(databaseURI)
+	if err != nil {
+		return err
+	}
+	defer db.Close()
+
+	dbURL, err := url.Parse(databaseURI)
+	if err != nil {
+		return err
+	}
+	switch dbURL.Scheme {
+	case "postgres":
+		rows, err := db.Query("SELECT extname FROM pg_catalog.pg_extension")
+		defer rows.Close()
+		Expect(err).ToNot(HaveOccurred())
+		extensions := []string{}
+		for rows.Next() {
+			var name string
+			err = rows.Scan(&name)
+			Expect(err).ToNot(HaveOccurred())
+			extensions = append(extensions, name)
+		}
+		Expect(rows.Err()).ToNot(HaveOccurred())
+		Expect(extensions).To(ContainElement("uuid-ossp"))
+		Expect(extensions).To(ContainElement("postgis"))
```

Comment:
@henrytk It worries me a little that a typo could disable this test and we might never notice. Chris suggested adding a `case "mysql"` that does nothing and then a `default` case which fails, so that if a typo happened the tests would fail. What do you think?